packages/app/src/components/Blank.jsx

```diff
@@ -173,7 +173,7 @@ export const sessionLifecycle = () => {
     <div class='container'>
       ${svgCy}
       <br/>
-      <p class="warn">Because <code><b>experimentalSessionSupport</b></code> is enabled, Cypress navigates to the default blank page <span class="em">before each test</span> to ensure test reliability.</p>
+      <p class="warn">Because <code><b>experimentalSessionAndOrigin</b></code> is enabled, Cypress navigates to the default blank page <span class="em">before each test</span> to ensure test reliability.</p>
```

Comment:
original in [runner-shared](https://github.com/cypress-io/cypress/pull/18075/files#diff-99266ff449fcce661698daa2bcad6bc500c25f2dcc50fafe79e966d8cfad63fcL214) in `9.x`